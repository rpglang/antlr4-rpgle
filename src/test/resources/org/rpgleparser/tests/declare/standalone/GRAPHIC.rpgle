        DCL-S firstName GRAPH(20);
        DCL-S fullName VARGRAPH(50);