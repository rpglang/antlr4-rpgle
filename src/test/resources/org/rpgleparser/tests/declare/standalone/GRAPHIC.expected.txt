/*===TOKENS===*/
DS_Standalone            <DCL-S>
ID                       <firstName>
GRAPH                    <GRAPH>
'('                      <(>
NUMBER                   <20>
')'                      <)>
FREE_SEMI                <;>
DS_Standalone            <DCL-S>
ID                       <fullName>
VARGRAPH                 <VARGRAPH>
'('                      <(>
NUMBER                   <50>
')'                      <)>
FREE_SEMI                <;>
/*===TREE===*/
(r
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword firstName)))
      (datatype (datatypeName GRAPH) (args ( (expression (number 20)) )))
      ;
    )
  )
  (statement
    (dspec
      DCL-S
      (identifier (free_identifier (idOrKeyword fullName)))
      (datatype (datatypeName VARGRAPH) (args ( (expression (number 50)) )))
      ;
    )
  )
)
/*======*/